spring.application.name=changebook_users

spring.datasource.url=jdbc:mysql://35.198.4.236:3306/change_book

spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5InnoDBDialect

# So app starts despite "table already exists" errors.
spring.datasource.continue-on-error=true

# Enforces database initialization
spring.datasource.initialization-mode=always

spring.datasource.username=root
spring.datasource.password=pucxyx-9pUgty-pycgoz


spring.datasource.hikari.connectionTimeout=20000
spring.datasource.hikari.maximumPoolSize=5

spring.jpa.properties.hibernate.default_schema=PUBLIC
spring.jpa.show-sql=true
server.port=8080

#Application
message-locator.service-name=${message_locator_service_name:${spring.application.name}}
message-locator.service-id=${message_locator_service_id:1}
jwt.secret=ChangeBookAuth

#spring.jpa.hibernate.ddl-auto=update
